### PR TITLE
Add overlay for Galaxy A30

### PR DESCRIPTION
**Pull request by crazo7924**
tests results are ok

### CODE REVIEW

**Comment by crazo7924** [comment]
@phhusson how to deal with corners padding?

**Comment by phhusson** [comment]
At the moment, I recommend users to set them in treble settings.
I know that's not ideal but well

**Comment by crazo7924** [comment]
@phhusson please merge

File: Samsung/A30/res/values/arrays.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="config_autoBrightnessDisplayValuesNits" />
```
[the remark this answers is not in the document]
Comment (crazo7924) [reply Samsung/A30/res/values/arrays.xml]: Pushed necessary changes

**Pull request merged**